sklearn/ensemble/_hist_gradient_boosting/tests/test_loss.py

```diff
@@ -91,14 +91,14 @@ def fprime2(x: np.ndarray) -> np.ndarray:
         return get_hessians(y_true, x)
 
     optimum = newton(func, x0=x0, fprime=fprime, fprime2=fprime2,
-                     maxiter=70, tol=5e-6)
+                     maxiter=70, tol=5e-7)
```

Comment:
Interesting. I am not sure what we should conclude from the fact that more newton iteration converge to the true optimum even with a lower precision gradient computation...